src/config/main.ejs

```diff
@@ -446,5 +446,15 @@ module.exports = {
         acceptSslCerts: true
       }
     }<% }} %>
+  },
+
+  usage_analytics: {
+    <% if (answers.allowAnonymousMetrics === undefined) { %>
+    enabled: true,
```

Comment:
Isn't it supposed to be opt-in? In that case, default should be `false`?

Reply:
It was decided that, in the case of `npm init nightwatch -y` we will set it to false. 
But when questions are asked we will default to true. Made changes accordingly.